### PR TITLE
Add new example

### PR DESCRIPTION
**Pull request by Zaidtech**
@andrefpoliveira, I add a new LinearSearch Example, kindly review it.

### CODE REVIEW

File: examples/LinearSearch/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# LINEAR SEARCH
+
+Arguments:  
+```Argument #1 - Type: Integer```
```
Comment (andrefpoliveira) [inline examples/LinearSearch/README.md:4]: First of all thanks for contributing! 😄🎉 
I belive this should be List right? 😉

**Review by andrefpoliveira — CHANGES_REQUESTED** [review]

File: examples/LinearSearch/README.md (hunk 1 above)
Comment (Zaidtech) [reply examples/LinearSearch/README.md:4]: haha, more are there , @andrefpoliveira  no  "sir" this time, haha..
If you allow more ideas are .. related to networking like my_ip .. port .. !!

**Review by andrefpoliveira — APPROVED** [review]

**Comment by Zaidtech** [comment]
Thanks, for merging, does it also needs a Hactober-fest checked label, 
This will be my first tee :wink:

**Comment by andrefpoliveira** [comment]
@Zaidtech sure! I forgot :)

**Comment by Zaidtech** [comment]
Thanks @andrefpoliveira, can the previous merged PR's be labeled in the same way.. 
Thanks again!!!

**Pull request merged**